crates/download/src/main.rs

```diff
@@ -186,6 +191,15 @@ async fn main() -> anyhow::Result<()> {
     )?);
 
     if !min {
+        let model_dir = output.join("model");
+        fs_err::create_dir(&model_dir)?;
+        tasks.spawn(download_and_extract_from_gh(
+            model,
+            Stripping::FirstDir,
+            &model_dir,
```

Comment:
中で(プログレスバーで管理されるasyncの文脈で)mkdirするようになっているので、ここでは必要無いと思います。
(あとこの時点で`output`が存在するとは限らないので、`create_dir`だと駄目で`create_dir_all`にする必要がある)

<https://github.com/VOICEVOX/voicevox_core/blob/22b46cd8b8d1bf95a0335b3f29802da434bf7674/crates/download/src/main.rs#L513-L515>

```suggestion
        tasks.spawn(download_and_extract_from_gh(
            model,
            Stripping::FirstDir,
            &output.join("model"),
```

Reply:
あっと本当ですね！ちょっとちゃんと確認してませんでした！